contrib/cindercompatibleapi/api/router.go

```diff
@@ -46,60 +48,29 @@ type ErrorSpec struct {
 
 // Run ...
 func Run(CinderEndPoint string) {
-	ep, ok := os.LookupEnv(c.OpensdsEndpoint)
+	OpensdsEP, ok := os.LookupEnv(c.OpensdsEndpoint)
```

Comment:
Don't define variable like this, I think `edp` is better.

Reply:
I actually prefer variable name to be spelled out so I can tell what it is.  What about "opensdsEndpoint".

Reply:
It looks good to me.